lua/entities/starfall_cnextbot.lua

```diff
@@ -0,0 +1,156 @@
+AddCSLuaFile()
+
+ENT.Base 			= "base_nextbot"
+ENT.Spawnable		= true
+
+
+-- Had to do some fuckery to get model to set on both realms when the bot is created from a chip
+function ENT:SetupDataTables()
+	self:NetworkVar( "String", 0, "NBModel" )
+end
+
+function ENT:Initialize()
+	if SERVER then
+		self.RagdollOnDeath = true
+		self.WALKACT = ACT_WALk
+		self.RUNACT = ACT_RUN
+		self.IDLEACT = ACT_IDLE
+		self.MoveSpeed = 200
+		
+		self.DeathCallbacks = SF.HookTable()
+		self.InjuredCallbacks = SF.HookTable()
+		self.LandCallbacks = SF.HookTable()
+		self.JumpCallbacks = SF.HookTable()
+		self.IgniteCallbacks = SF.HookTable()
+		self.NavChangeCallbacks = SF.HookTable()
+	end
+
+	self:SetModel( self:GetNBModel() )
```

Comment:
I don't understand why the client needs to set the model. Only the server should have to do that.

Reply:
IIRC I had some problems with the model not setting when set just on Server. Maybe a SP or issue or maybe I did something wrong. I'll play around with it after I do the other stuff.